packages/veto-app/frontend/src/layouts/MainLayout/MainLayout.tsx

```diff
@@ -0,0 +1,45 @@
+import React, { FunctionComponent, useState } from 'react'
+import { Flex } from '@veriphi/veto-ui'
+import { Router } from '@reach/router'
+import Dashboard from 'pages/Dashboard'
+import Wallets from 'pages/Wallets'
+import History from 'pages/History'
+import Explorer from 'pages/Explorer'
+import Monitoring from 'pages/Monitoring'
+import SideMenu from '../../components/molecules/SideMenu'
+
+export type MainLayoutLabel = 'Dashboard' | 'Wallets' | 'History' | 'Explorer' | 'Monitoring'
+export type MainLayoutRoute = '/dashboard' | '/wallets' | '/history' | '/explorer' | '/monitoring'
+
+type Props = {
+  path: string
+}
+
+const getActiveMenu = (currentPathName: MainLayoutRoute | string): MainLayoutRoute => {
+  // https://www.typescriptlang.org/docs/handbook/advanced-types.html#using-type-predicates
+  const isMainLayoutRoute = (route: MainLayoutRoute | string): route is MainLayoutRoute =>
```

Comment:
Move outside of `getActiveMenu` to not re-reate the function on each execution.

Great usage of https://www.typescriptlang.org/docs/handbook/advanced-types.html#user-defined-type-guards btw :O :+1: